packages/ui-components/src/button/Button.types.ts

```diff
@@ -0,0 +1,10 @@
+import { HTMLAttributes } from 'react';
+
+export interface ButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  buttonAction: React.MouseEventHandler<HTMLButtonElement>;
+  disabled: boolean;
+  label: string;
+  size: 'large' | 'medium' | 'small';
+  variant: 'primary' | 'secondary';
```

Comment:
use enums for this and export them as well. so that size can be used as 

size={Button.SIZE.LARGE}
and same for variant